DataManagementSystem/Client/FTSRequest.py

```diff
@@ -796,7 +793,7 @@ def __resolveFTSServer( self ):
 
     targetSite = ''
     for targetSite in targetSites:
-      targetFTS = getFTSServersForSites( [targetSite] )
+      targetFTS = getFTS2ServersForSites( [targetSite] )
```

Comment:
Uncool with FTS3... this FTSRequest is an ugly stuff... the FTS system should be redesigned to avoid code duplication like we have here. But for the time being, one needs to check the FTSVersion variable in Operation ...